STAGES.md

```diff
@@ -57,10 +58,9 @@ to a particular way of solving a problem.
 
 > “This idea is becoming part of the web.”
 
-A **Candidate Recommendation** [championed] by a [W3C Working Group], usually
-implemented by at least 2 [recognized browser vendors], possibly behind a flag. |
-It should be considered stable and subject to little change. Stage 3 features
-will likely become a standard.
+A **Candidate Recommendation** [championed] by a [W3C Working Group] or implemented by at least 3 [recognized browser vendors], possibly behind a flag. It should be considered stable and subject to little change. Stage 3 features
```

Comment:
Re-reading everything, I am also unsure if this change is correct.
There is also the `or` here between spec status and implementation.

But the requirement has been updated from 2 to 3 vendors. Is this correct?

If its implemented by 3 vendors it is already de-facto part of the web.

Reply:
See : https://github.com/csstools/cssdb/pull/72#discussion_r777661072